spec/emittr_spec.rb

```diff
@@ -103,6 +103,85 @@
     end
   end
 
+  describe '#on_any' do
+    let(:block)     { Proc.new {} }
+    let(:any_block) { Proc.new {} }
+
+    context 'when #emit is called' do
+      it 'calls #on_any blocks' do
+        emitter.on_any &any_block
+
+        emitter.on :first_any_test, &block
+        emitter.on :second_any_test, &block
+
+        expect(any_block).to receive(:call).exactly(4).times
```

Comment:
got it. I'll need to change the `#emit_any` method, so it can call `#any` blocks passing the emitted `event`